e2e/__tests__/jestChangedFiles.test.ts

```diff
@@ -91,7 +100,8 @@ testIfHg('gets hg SCM roots and dedupes them', async () => {
   );
 });
 
-test('gets git SCM roots and dedupes them', async () => {
```

Comment:
this test is changed instead of copied and tweaked?

Reply:
sorry this diff is hard to read!  This test is still here but moved down.

I wanted to put each relevant test near each other, but it makes this diff hard to read.  Would you rather I put all new tests at the bottom of this file?

Reply:
I'm all for better grouping! 👍 could you send a separate PR reorganising the tests in such a way that the diff in _this_ PR is additive/easier to read?